include/Options/Business/Domains.php

```diff
@@ -124,6 +132,30 @@ protected function sanitize( $value, Options $options ) {
 			}
 		}
 
+		// Detect invalid language slugs.
+		if ( ! empty( $value ) ) {
+			// Non-blocking error.
+			$value = array_keys( $value );
+
+			$this->errors->add(
```

Comment:
What about one error per unknown language?

Reply:
Isn't it better to regroup all unknown languages in a single message?